geomesa-web/geomesa-web-core/src/main/scala/org/locationtech/geomesa/web/core/SpringScalatraBootstrap.scala

```diff
@@ -11,16 +11,21 @@ package org.locationtech.geomesa.web.core
 import javax.servlet.ServletContext
 import javax.servlet.http.{HttpServletRequest, HttpServletRequestWrapper, HttpServletResponse}
 
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.commons.lang.exception.ExceptionUtils
 import org.locationtech.geomesa.accumulo.data.AccumuloDataStoreFactory
-import org.scalatra.ScalatraServlet
 import org.scalatra.servlet.RichServletContext
+import org.scalatra.{InternalServerError, ScalatraServlet}
 import org.springframework.context.{ApplicationContext, ApplicationContextAware}
 import org.springframework.web.context.ServletContextAware
 
 import scala.beans.BeanProperty
 import scala.collection.JavaConversions._
 
-trait GeoMesaScalatraServlet extends ScalatraServlet {
+trait GeoMesaScalatraServlet extends ScalatraServlet with Logging {
```

Comment:
That link says scala 2.11 doesn't support java 8 (or it's experimental)... so whatever issues there are would affect everything, not just scalatra.
I think @jahhulbert-ccri is right, that we can run on java 8 but not compile with it. I think Alec has done that in places...